image.go

```diff
@@ -103,3 +165,53 @@ func MergeImageConfig(dst *DockerImageConfig, src *ImageConfig) error {
 
 	return nil
 }
+
+func getBaseOutputImage(spec *Spec, target, defaultBase string) string {
+	baseRef := defaultBase
+	if spec.Targets[target].Image != nil && spec.Targets[target].Image.Base != "" {
+		baseRef = spec.Targets[target].Image.Base
+	}
+	return baseRef
+}
+
+func mergeSpecImage(spec *Spec, target string) *ImageConfig {
+	var cfg ImageConfig
+
+	if spec.Image != nil {
+		cfg = *spec.Image
+	}
+
+	if i := spec.Targets[target].Image; i != nil {
+		if i.Entrypoint != "" {
+			cfg.Entrypoint = spec.Targets[target].Image.Entrypoint
+		}
+
+		if i.Cmd != "" {
+			cfg.Cmd = spec.Targets[target].Image.Cmd
+		}
```

Comment:
nit: keep consistent the the value setting for these
```suggestion
		if i.Entrypoint != "" {
			cfg.Entrypoint = i.Entrypoint
		}

		if i.Cmd != "" {
			cfg.Cmd = i.Cmd
		}
```

Reply:
done